### PR TITLE
feat: add hotkeys to the app

### LINKED ISSUE #985: feat: add hotkeys to the app
# 🚀 Feature Request

### 📝 Description

Add hotkeys for the various features of the app, and show the hotkey commands for those features in the UI as well.

### PR DESCRIPTION
**Pull request by themohammadsa**
# ✨ Pull Request

### 📓 Referenced Issue
Fixes #985 

### ℹ️ About the PR
Hotkeys have been added for various buttons in the application.

A shortcut button has been added to the toolbar. Clicking this button will open a modal displaying the complete list of available shortcuts for the application.

🖼️ Testing Scenarios / Screenshots

Screenshot 2023-06-22 at 6 14 48 PM

### CODE REVIEW

File: desktop-app/src/main/menu/view.ts [hunk 1]
```diff
@@ -32,9 +32,9 @@ const getReloadMenu = (
   click: () => {
     if (isDev) {
       mainWindow.webContents.reload();
-      return;
+      // return;
     }
-    mainWindow.webContents.send('reload', {});
+    // mainWindow.webContents.send('reload', {});
```
Comment (manojVivek) [inline desktop-app/src/main/menu/view.ts:37]: Why reason why this is commented out?

**Review by manojVivek — APPROVED** [review]
Just a couple of minor things, looks so neat otherwise! 👏🏼

File: desktop-app/src/main/menu/view.ts (hunk 1 above)
Comment (manojVivek) [reply desktop-app/src/main/menu/view.ts:37]: Ok, but if we have this commented then the menu option, won't work, I guess? 
The `View` menu -> `Reload` option.

Comment (themohammadsa) [reply desktop-app/src/main/menu/view.ts:37]: Oops! Apologies for missing that. I will make the necessary changes. 😅

**Pull request merged**